app/containers/TrackInfo/index.js

```diff
@@ -128,10 +128,10 @@ export function TrackInfo(props) {
   }
 
   useEffect(() => {
-    if (!checkTrackID(trackData.trackId)) {
+    if (!InfoLoading && !checkTrackID(trackData.trackId)) {
```

Comment:
why is the variable name Pascal Case? Generally we use Pascal Case for class and components only.